### ISSUE TITLE
tmux; lynx; pstree; mc; man

### ISSUE THREAD

**Issue by pflegende** [webinstall/webi-installers#197]
often playing around on jupyterLabs;

sometimes it would be practical to have a splittet screen and also the great attach / deattach property ...

but int the binder environment  it means always to build a new docker for only one single step ...

short to say:
_would be very nice_

**Comment by coolaj86** [comment]
Can you get the links for the release APIs of each of those?

**Comment by coolaj86** [comment]
@pflegende Can you get the links for the release APIs of each of those?

**Comment by coolaj86** [comment]
Not having heard anything back, I'm going to close this.

I'm not opposed to having these if someone else does the work, but I'm not going to work on it right now because those aren't modern cross-platform tools that really fit with the goal of Webi.

**Comment by coolaj86** [comment]
P.S. Have you tried `screen`? Or [`lf`](https://webinstall.dev/lf/) (like `mc`, I think, but cross-platform)?

**Comment by pflegende** [comment]
excuse me for have you waiting for me.
Thank you for the proposals.
I know screen, will check it out in the destination environment (jupyter).
Switched for me from screen to tmux because of the remote capabilities in tmux.
But may be, screen can give us here enough good new experience in the jupyter environment.

And lf is very cool thing. Thank you for this.
I tried out also with webinstall in jupyter: icredible!

So we found it cozy, to work also with some of the listed stuff ...
:)

... and the best way to bring it in a running jupyter environment seems your great webinstall.
But dont feel compelled to leave your roadmap, I highly apreciate to follow a cool shedule.

thank you
